modules/persistent-storage-vsphere-static-provisioning.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-static-provisioning_{context}"]
+= Statically provisioning VMware vSphere volumes
+
+To statically provision VMware vSphere volumes you must create the virtual machine disks for reference by the persistent volume framework.
+
+.Prerequisites
+
+* Storage must exist in the underlying infrastructure before it can be mounted as a volume in {product-title}.
+
+== Create the virtual machine disks
+
+Virtual machine disks (VMDKs) must be created manually before statically provisioning VMware vSphere volumes. Use either of the following methods:
+
+* Create using `vmkfstools`. Access ESX through Secure Shell (SSH) and then use following command to create a VMDK volume:
++
+----
+$ vmkfstools -c <size> /vmfs/volumes/DatastoreName/volumes/<disk-name>.vmdk
+----
+
+* Create using `vmware-diskmanager`:
++
+----
+$ shell vmware-vdiskmanager -c -t 0 -s <size> -a lsilogic <disk-name>.vmdk
+----
+
+== Create the PersistentVolume
+
+.Procedure
+
+. Create a file, `pv.yaml`, with the PersistentVolume object definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv <1>
+spec:
+  capacity:
+    storage: 2Gi <2>
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  vsphereVolume: <3>
+    volumePath: "[datastore1] volumes/myDisk"  <4>
+    fsType: ext4  <5>
+----
+<1> The name of the volume. This name is how it will be identified by PersistentVolumeClaims or Pods.
+<2> The amount of storage allocated to this volume.
+<3> The volume type being used. This example uses `vsphereVolume`. The label is used to mount a vSphere VMDK volume into pods. The contents of a volume are preserved when it is unmounted. The volume type supports VMFS and VSAN datastore.
+<4> The existing VMDK volume to use. You must enclose the datastore name in square brackets, `[]`, in the volume definition, as shown above.
```

Comment:
~above~
> ! above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”